src/Styles/index.css

```diff
@@ -0,0 +1,92 @@
+@import "./star.css";
+@import "./button.css";
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
```

Comment:
And styles with the prefix `card__` should be kept in a file named `card.css` (as of BEM)